packages/swing-store/test/test-snapstore.js

```diff
@@ -36,30 +34,40 @@ test('build temp file; compress to cache file', async t => {
     'ba7816bf8f01cfea414140de5dae2223b00361a396177a9cb410ff61f20015ad';
   t.like(result, {
     hash: expectedHash,
-    filePath: path.resolve(pool.name, `${expectedHash}.gz`),
     rawByteCount: 3,
     rawSaveSeconds: 0,
     compressSeconds: 0,
   });
-  t.is(await store.has(hash), true);
+  const snapshotInfo = store.getSnapshotInfo('fakeVatID');
+  t.deepEqual(snapshotInfo, {
+    endPos: 47,
+    hash: expectedHash,
+    uncompressedSize: 3,
+    compressedSize: 23,
```

Comment:
haha "compression"

Reply:
Compression gets very rubbery near the bottom.